### ISSUE TITLE
Screen reader gets stuck in infinite loop on Tock Report Period

### ISSUE THREAD

**Issue by cantsin** [18F/tock#1727]
(Copying over https://github.com/18F/TLC-crew/issues/198 to here)

> ## Brief description
> Navigating the Tock Reporting Period page with the combination of VoiceOver and Safari puts the user into an infinite loop. The user is unable to make a Tock submission.
> 
> ## Standards
> * [4.1.1 Parsing (WCAG2.0 A)](http://www.w3.org/TR/2008/REC-WCAG20-20081211/#ensure-compat-parses)
> 
> 
> ## Steps to Reproduce
> While using the combination of VoiceOver and Safari, navigate to the Tock Reporting Period page (https://tock.18f.gov/reporting_period/XXXX/).
> 
> ### Expected
> User is able to navigate to the Tock Report Period page and submit a Tock time report.
> 
> ### Actual
> User gets stuck in an infinite loop.
> 
> ***
> ## Acceptance criteria
> - [ ] User can navigate and interact with the page using the screen reader combination of VoiceOver and Firefox.
